hls4ml/backends/quartus/passes/recurrent_templates.py

```diff
@@ -301,5 +306,9 @@ def __init__(self):
 
     def format(self, node):
         params = self._default_function_params(node)
-        params['weights'] = 'w{0}, wr{0}, b{0}'.format(str(node.index))
+        if "pytorch" in node.attributes.keys():
```

Comment:
Is it better to check that pytorch is in the keys or that it is defined and True? What if it is defined and set False? I wonder if using `node.get_attr('pytorch')` (returns `None` if not found) or `node.get_attr('pytorch', False)` is better.

Reply:
Right now it will only be defined if a recurring layer is parsed in pytorch, and I can't really envision a situation where we would have code that sets this key that isn't part of the pytorch parser. But I still think you are right that this should be implemented in a more future-proof way and `node.get_attr('pytorch', False)` is probably the most stringent solution. I will implement it.